doc/api/cli.md

```diff
@@ -1609,6 +1576,21 @@ added: v0.8.0
 
 Silence deprecation warnings.
 
+### `--no-experimental-detect-module`
```

Comment:
How is it on by default if it is experimental? Perhaps `--no-detect-module`?

Reply:
We have lots of stuff experimental yet on by default. -1 on changing the flag, we want to keep `--experimental-detect-module` as a no-op anyway, the flag _is_ experimental, and we plan to remove it without going through a deprecation cycle, which is another good reason to keep "experimental" in its name.